### PR TITLE
Devel gurling

### PR DESCRIPTION
**Pull request by vDingus**
# Description

Simple script to automate Polaris EC2 Native on demand snapshots from inside an EC2 instance

## Related Issue

NA

NA

## Motivation and Context

customer request

## How Has This Been Tested?

Tested on an EC2 instance protected by Rubrik Polaris.

## Screenshots (if appropriate):

image

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply. If you're unsure about any of these, don't hesitate to ask. We're here to help!

- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **[CONTRIBUTION](https://github.com/rubrikinc/welcome-to-rubrik-build/blob/master/CONTRIBUTING.md)** document.
- [ ] I have updated the CHANGELOG file accordingly for the version that this merge modifies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+* `POLARIS_PASSWORD`
 
-## :mag: Example
+The values of `POLARIS_SUBDOMAIN`, `USERNAME`, and `PASSWORD` can also be assigned in *polaris-ec2native-ondemand.py* if you are retrieving from a secret store programatically somehow.
 
-## :blue_book: Documentation
+Ensure you set `snapcount` equal to the number of snapshots you want to retain, and enable or disable logging by setting `logging` equal to True or False in *polaris-ec2native-ondemand.py*
 
-Here are some resources to get you started! If you find any challenges from this project are not properly documented or are unclear, please raise an issueand let us know! This is a fun, safe environment - don't worry if you're a GitHub newbie! :heart:
+Once configured, run this script using Python 3.6+ from an EC2 instance being protected by Polaris. Each time it is run, it will take an on demand snapshot of this instnace and expire (starting with the oldest) any on demand snapshots this instance in excess of `snapcount`.
```
Comment (drew-russell) [inline README.md]: `instnace` typo

Comment (drew-russell) [reply README.md]: `on demand snapshot of this instnace and expire`

Comment (DamaniN) [reply README.md]: I would update the "How tested" section with a slightly more detailed explanation. Public reviewers may not know this test environment.

File: polaris-ec2native-ondemand.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+# USE - Takes an on demand snapshot of the EC2 instance running this script
+# Deletes oldest on demand snapshots of this ec2 instance if # of on demand snapshots exceeds snapcount
```
Comment (DamaniN) [inline polaris-ec2native-ondemand.py]: Third comment line wrapped around. 

Also, mention in the `USE -` section that this script is used to take Polaris ON-Demand snapshots.

File: README.md (hunk 1 above)
Comment (vDingus) [reply README.md]: fixed typo and added more detail as requested

**Review by DamaniN — CHANGES_REQUESTED** [review]

**Review by DamaniN — DISMISSED** [review]

File: polaris-ec2native-ondemand.py (hunk 2 above)
Comment (vDingus) [reply polaris-ec2native-ondemand.py]: added requested detail to USE section

**Review by drew-russell — APPROVED** [review]

**Pull request merged**